docs/Developer Tools/Configuring SDL Logging/index.md

```diff
@@ -59,19 +59,21 @@ Targets are the output locations where the log will appear. By default, in both
 The Apple System Logger target, `SDLLogTargetAppleSystemLogger`, is the default log target for both default and debug configurations on devices running iOS 9 or older. This will log to the Xcode console and the device console.
 
 #### OS Log Target
-The OSLog target, `SDLLogTargetOSLog`, is the default log target in both default and debug configurations for devices running iOS 10 or newer. For more information on this logging system see [Apple's documentation](https://developer.apple.com/reference/os/logging). SDL's OSLog target will take advantage of subsystems and levels to allow you powerful runtime filtering capabilities through MacOS Sierra's Console app with a connected device.
+The OSLog target, `SDLLogTargetOSLog`, is the default log target in both default and debug configurations for devices running iOS 10 or newer. For more information on this logging system see [Apple's documentation](https://developer.apple.com/reference/os/logging). SDL's OSLog target will take advantage of subsystems and levels to allow you powerful runtime filtering capabilities through the MacOS Console app with a connected device.
 
 #### File Target
-The File target, `SDLLogTargetFile`, allows you to log messages to a rolling set of files (default 3) which will be stored on the device, specifically in the `Documents/smartdevicelink/log/` folder. The file names will be timestamped with the start time.
+The File target, `SDLLogTargetFile`, allows you to log messages to a rolling set of files which will be stored on the device, specifically in the `Documents/smartdevicelink/log/` folder. The file names will be timestamped with the start time.
 
-To access the file, you can either access it from runtime on the device (for example, to attach it to an email that the user sends), or if you have access to the device, you can access them via iTunes as well with small modifications to your app:
+To access the file, you can either access it from runtime on the device (for example, to attach it to an email that the user sends), or if you have access to the device, you can access them via the MacOS Finder. To access the files on the device you must make the following small modifications to your app:
 
 1. Add the key `UIFileSharingEnabled` to your `info.plist`. Set the value to `YES`.
-2. Connect the device to a computer that has iTunes installed.
-3. Open iTunes, click on the icon for the device, then click on "File Sharing" > "(Your App Name)"
-4. You should see a folder called "smartdevicelink". Select the file and click save. When you open the folder on your computer, you will see the log files for each session (default maxes out at 3).
+1. Connect the device to a MacOS computer.
+1. Open the Finder, click on the device in the sidebar, then click on "Files" > "(Your App Name)"
```

Comment:
These steps require MacOS Catalina, but developers can still be on Mojave and use the latest Xcode.

Reply:
Fixed. Added instructions for both.